### ISSUE TITLE
fade 

### ISSUE THREAD

**Issue by twentyfortysix** [noelboss/featherlight#187]
Hello,

what about real image fading, instead of fade to white and shoot in an image?
Maybe some parameters as well, like speed, type etc..
.)

**Comment by ericmuigai** [comment]
Hey twentyfortysix,
Did you ever find the solution to this?

**Comment by twentyfortysix** [comment]
nope.. I just used something else I guess. But my information is not up to date, so it might have changed.

**Comment by ericmuigai** [comment]
Still the same. What did you end using?

**Comment by twentyfortysix** [comment]
I'm not sure.. I mostly use http://fancybox.net/ 
or if I want to build something "on my own" I use http://kenwheeler.github.io/slick/ I know it is a carousel, but it has the fade mode and if you are able to write the modal your self and hack in the carousel, it might be it.
